paths.py

```diff
@@ -1,5 +1,6 @@
 
 # Windows 10 EWDK (can just be ISO mount point)
```

Comment:
Your changes to this file are related to your local build environment, so it would be best not to include them here.